### PR TITLE
fix link in old docs to new docs

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: _layouts/fluo-doc.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <div class="fluo-doc">
 
   {% if page.version != site.latest_fluo_release %}
```
Comment (keith-turner) [inline _layouts/fluo-doc.html]: @mikewalch  should the check here change?

Comment (mikewalch) [reply _layouts/fluo-doc.html]: That if statement can be removed as all documentation using this layout is now old.

**Review by mikewalch — APPROVED** [review]

File: _layouts/fluo-doc.html (hunk 1 above)
Comment (keith-turner) [reply _layouts/fluo-doc.html]: ok I removed it, are the updates ok?

Comment (mikewalch) [reply _layouts/fluo-doc.html]: yes

**Pull request merged**